src/libs/xinterface/include/storm/xinterface/options_parser.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "attributes.h"
+
+#include <string>
+
+namespace storm {
+
+namespace detail {
```

Comment:
To my knowledge, `detail` meant to hide impl details of the code inlined into headers (templates or h-only libraries). Here it is used as a self-sufficient code, therefore I would rename that namespace to smth like`util`.
I checked our `util` lib and there is also `detail` instead of `util` that I would also rename...

But naming is not a big problem, and everything else seems just good to me. So anyway, I just want to know your opinion, @Hammie

Reply:
I like to put things in `detail` that should not be used in user code, but for some reason need to be declared/defined in a header. In this case I forgot to remove it as `removeCarriageReturn` is being used (and should be), so I suggest just removing `detail` here and leaving it in the `storm` namespace?

I would leave the ones in `util` though, as those are not intended to be used for anything else.